### ISSUE TITLE
Interactive markers should be `role="button"`

### ISSUE THREAD

**Issue by Malvoz** [mapbox/mapbox-gl-js#11031]
Interactive markers (e.g. markers that bind to popups) are focusable and actionable elements, and should have appropriate semantics (i.e. [`role="button"`](https://www.w3.org/TR/wai-aria/#button)).

Related WCAG 2.1 Success Criterion: [4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value) (this issue is to address the <q>Role</q> part of the Success Criterion for interactive markers).

**Comment by DillonKyleDev** [comment]
Hello, @mourner and @Malvoz!

I would love to help contribute to this project!  I wanted to fork and clone the repo to get a feel for the code base before commenting on a specific issue, but I'm facing some difficulties getting the project up and running locally.
I am stuck at the step:
>Install headless-gl dependencies https://github.com/stackgl/headless-gl#windows

In vsCode when I enter:
`copy node_modules/headless-gl/deps/windows/dll/x64/*.dll c:\windows\system32`

I get the error:
`copy : Cannot find path 'C:\Users\dillo\gitRepos\Mapbox\mapbox-gl-js\node_modules\headless-gl\deps\windows\dll\x64' because it does not exist.`

I believe I have all the packages installed correctly, any insight would be greatly appreciated!
